integration/token/fungible/views/auditor.go

```diff
@@ -148,3 +149,79 @@ func (p *RegisterAuditorViewFactory) NewView(in []byte) (view.View, error) {
 	f := &RegisterAuditorView{}
 	return f, nil
 }
+
+type CurrentHolding struct {
+	EnrollmentID string `json:"enrollment_id"`
+	TokenType    string `json:"token_type"`
+}
```

Comment:
It seems that `CurrentHolding` and `CurrentSpending` contain the same fields. Will we add more fields in the future which will make these two types different? Otherwise can't we use a common struct?

Reply:
Also, as this is part of the `views` package, the usage of this type feels not intuitive for me. 
```go
	spendingBoxed, err := network.Client("auditor").CallView("spending", common.JSONMarshall(&views.CurrentSpending{
		EnrollmentID: eID,
		TokenType:    typ,
	}))
```

This type is used to serialize the input of the view, so should we give it a name like `SpendingRequest`, `SpendingInput`, or similar?

Reply:
If we are to merge the structs, we need to come up with a better name. This struct corresponds to a filter?

Reply:
That struct just contain the input to the view.
```In the current integration tests, we use struct X for the input to a view, XView for the view, XViewFactory for the factory. So, they all have the same prefix.```.
Do we still need to change the name?